_studio/shared/src/libmfx_core_vaapi.cpp

```diff
@@ -249,7 +249,7 @@ typedef struct {
 
     /* CFL, GT information can be wrong, need to verify */
     { 0x3E90, MFX_HW_CFL, MFX_GT1 },
-    { 0x3E91, MFX_HW_CFL, MFX_GT2 },
+    { 0x3E91, MFX_HW_CFL, MFX_GT1 },
```

Comment:
Part of the driver believes that's GT2 and part believes that's GT1:
```
[dvrogozh@dvrscl media-driver]$ fgrep -rsn 0x3e91 | grep RegisterDevice
media_driver/linux/gen9/ddi/media_sysinfo_g9.cpp:814:    RegisterDevice(0x3e91, &cflGt1Info);
[dvrogozh@dvrscl media-driver]$ fgrep -rsn 0x3E91
media_driver/linux/common/os/i915/include/intel_chipset.h:223:#define PCI_CHIP_COFFEELAKE_S_GT2_1     0x3E91
```

Reply:
Mesa also thinks that's GT2:
[dvrogozh@dvrscl mesa]$ fgrep -rsn 0x3E91
include/pci_ids/i965_pci_ids.h:174:CHIPSET(0x3E91, cfl_gt2, "Intel(R) UHD Graphics 630 (Coffeelake 3x8 GT2)")

Reply:
And libdrm believes the same: https://cgit.freedesktop.org/mesa/drm/tree/intel/i915_pciids.h#n385

Reply:
Looking into the spec I believe that mesa and kernel is correct: that's GT2. File a bug against media-driver.